check_versions.sh

```diff
@@ -36,6 +38,6 @@ if $(version_gt $APP_USER_VERSION $LAST_VERSION); then
      echo "A release would update from $LAST_VERSION to $APP_USER_VERSION on Dockerhub."
      exit 0;
 else
-     echo "There is a problem in your versionning, the last version on the repository is $LAST_VERSION and you are trying to push the version $APP_USER_VERSION" >&2
+     echo "There is a problem in your versioning, the last version on the repository is $LAST_VERSION and you are trying to push the version $APP_USER_VERSION" >&2
```

Comment:
In the cookiecutter I added a new way of checking for both valid version number and whether the version exists on dockerhub [here](https://github.com/ITISFoundation/cookiecutter-osparc-service/blob/master/%7B%7Bcookiecutter.project_slug%7D%7D/Makefile)

And it looks like this:
```
version_valid = $(shell test $$(echo $(DOCKER_IMAGE_TAG) | cut --fields=1 --delimiter=.) -gt 0 > /dev/null && echo "image version is valid")
version_exists = $(shell DOCKER_CLI_EXPERIMENTAL=enabled docker manifest inspect $(DOCKER_REGISTRY)/$(DOCKER_IMAGE_NAME):$(DOCKER_IMAGE_TAG) > /dev/null && echo "image already exists on $(DOCKER_REGISTRY)")
push push-force: ## pushes (resp. force) services to the registry if service not available in registry.
	@$(if $(findstring force,$@),,\
		$(if $(call version_valid),$(info version is valid), $(error $(DOCKER_IMAGE_TAG) is not a valid version (major>=1)))\
		$(if $(call version_exists),$(error $(DOCKER_REGISTRY)/$(DOCKER_IMAGE_NAME):$(DOCKER_IMAGE_TAG) already exists on $(DOCKER_REGISTRY)), $(info no version found on $(DOCKER_REGISTRY)))\
	)
	@$(MAKE) push-version;
	@$(MAKE) push-latest;
```